packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -1141,16 +1141,7 @@ void main() {
   });
```

Comment:
@dishankjindal1 can you check the indentation from earlier comments? Some looks like 4 spaces, the flutter style guide prefers two. That may be what remains to resolve here.